src/index.html

```diff
@@ -5,10 +5,37 @@
     <meta name="viewport"
           content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500;700&display=swap" rel="stylesheet">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
   </head>
+  
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+
+      <nav class="nav">
+
+        <div class="icon-block"><img src="./images/logo.png" alt="MOYO logo" class="moyo-icon"></div>
+
+        <ul class="nav__list">
+
+          <li class="list__item"><a href="#" class="link is-active selected">Apple</a></li>
```

Comment:
the new tag must be on a new line, fix in all places
```suggestion
          <li class="list__item">
            <a href="#" class="link is-active selected">
              Apple
            </a>
          </li>
```